build-image-src/Dockerfile-python313

```diff
@@ -0,0 +1,58 @@
+ARG IMAGE_ARCH
+FROM public.ecr.aws/lambda/python:3.13-$IMAGE_ARCH
```

Comment:
Would the tests pass if we change this to preview? i.e. `public.ecr.aws/lambda/python:3.13-preview`

Reply:
I tried that with 1 of the previous [commits](https://github.com/aws/aws-sam-build-images/actions/runs/11667229455/job/32484071114) but since the init templates are not merged and lambda builders/sam cli are not released, 2 tests fail